pom.xml

```diff
@@ -15,6 +15,11 @@
     
     <repositories>
         <repository>
+          <id>central</id>
+          <name>central</name>
+          <url>https://repo1.maven.org/maven2</url>
+        </repository>
+        <repository>
```

Comment:
Kind of surprised we don't have this in the parent pom.

We do have a local (to UW) proxy to central at https://code.doit.wisc.edu/maven/content/repositories/central/